outputs.tf

```diff
@@ -5,10 +5,10 @@ output "required_tags_rule_arn" {
 
 output "aws_config_role_arn" {
   description = "The ARN of the AWS config role."
-  value       = aws_iam_role.main.arn
+  value       = concat(aws_iam_role.main.*.arn, [""])[0]
```

Comment:
Just double checking here: do you intend for the second item to be of length 1 or 0? I read somewhere that [] is length 0 and [""] is length 1.

Reply:
I believe it needs to be a length of 1 because outputs can't be conditionally enabled or disabled. So this is a workaround to make the output an empty string if the resource doesn't exist.

Reply:
Got it. That makes sense. 👍

Reply:
I would LOVE to know if there's a better way to do it.

Reply:
The logic seems sound and it appears to be the most succinct way of doing this. The only other thing that comes to mind would be something like:

```terraform
value = some-type-of-if-statement ? aws_iam_role.main.*.arn : ""
```

But then you're left with the `*` in that output, which I don't think will go over well since the output is intended to be a string.